tfe/resource_tfe_team_organization_member.go

```diff
@@ -132,3 +133,40 @@ func unpackTeamOrganizationMemberID(id string) (teamID, organizationMembershipID
 
 	return s[0], s[1], nil
 }
+
+func resourceTFETeamOrganizationMemberImporter(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	tfeClient := meta.(*tfe.Client)
+
+	// Import formats:
+	//  - <TEAM ID>/<ORGANIZATION MEMBERSHIP ID>
+	//  - <ORGANIZATION NAME>/<TEAM NAME>/<USER EMAIL>
+	s := strings.Split(d.Id(), "/")
```

Comment:
TIL email addresses can contain forward slashes!

Perhaps it would be better to use SplitN and let it fail on the email lookup if there are more than 3 partitions

Reply:
Shoot, I thought I had this working earlier with escaping the slashes, but looking again now it seems like it does not. Good catch!

> Perhaps it would be better to use SplitN and let it fail on the email lookup if there are more than 3 partitions

I like that suggestion, and am going to use it in [this similar PR](https://github.com/hashicorp/terraform-provider-tfe/pull/715/files#diff-1e092be463f14185a005a40ad7af1b58e0b9b257621d04641dd5c650620112ceR120) I have, but a potential problem is that team names can also have forward slashes (e.g. `team/with/slashes`)...so I could have an import ID like `org-name/team/with/slashes/user/name@email.com` which would partition into the the team being `team` and the email being `with/slashes/user/name@email.com`

I _think_ we would need a way to escape slashes in the emails and team names....maybe something [like this](https://codereview.stackexchange.com/a/259382)?

Or we could change the order of the ID to `<ORGANIZATION NAME>/<USER EMAIL>/<TEAM NAME>`, use SplitN, and accept that it is pretty unlikely to have both an email **and** a team name with slashes in them.

Reply:
That is trouble! I didn't realize that was true about team names, also. There's really not a good way to deal with this, and escaping doesn't feel so great because you'd need a double escape to overcome shell escaping, right?

Reply:
yah, and it would also be frustrating to have some imports (like this one) where you need to escape slashes, and others (where we can use SplitN) where you shouldnt

I wonder, then, if it is better to arrange the alternate import ID like `<ORGANIZATION NAME>/<USER EMAIL>/<TEAM NAME>` and `SplitN 3`...and just not support the rare case where you have slashes in your email...what do you think?